docs/stylesheets/extra.css

```
@@ -120,6 +315,114 @@
+     color to the top only — no "footer" bleed at the bottom. */
+  background: linear-gradient(
+    to bottom,
+    var(--md-primary-fg-color) 0,
+    var(--md-primary-fg-color) 1.9rem,
+    hsla(var(--md-hue), 6%, 18%, 1) 1.9rem,
+    hsla(var(--md-hue), 6%, 18%, 1) 100%
+  );
+  box-shadow:
+    0 0 24px rgba(2, 132, 199, 0.18),      /* sky glow */
+    0 8px 28px rgba(0, 0, 0, 0.18);        /* drop shadow */
+  overflow: hidden;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)